api/proto/info/info.proto

```diff
@@ -111,7 +111,7 @@ message ZInfoNetwork {
   repeated string IPAddrs = 5; // All IP addresses with /N for subnet
   repeated string defaultRouters = 6; // If DHCP assigned
   ZInfoDNS dns = 7; // If DHCP assigned
-  bool up = 8;      // operational up/down status.
+  bool ipv4Up = 8;      // operational up/down status.
```

Comment:
For new fields we conform with using _ instead of camelCase.
Thus ipv4_up.
The generated go code is the same thus you still access the field using ipv4Up.

Reply:
done